Tests/CSharp/CodeRefactorings/ContractEnsuresNotNullReturnTests.cs

```diff
@@ -0,0 +1,97 @@
+using System;
+using NUnit.Framework;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis;
+using System.Collections.Immutable;
+using RefactoringEssentials.CSharp.CodeRefactorings;
+using System.Diagnostics.Contracts;
+
+namespace RefactoringEssentials.Tests.CSharp.CodeRefactorings
+{
+    [TestFixture]
+    public class ContractEnsuresNotNullReturnTests : CSharpCodeRefactoringTestBase
+    {
+        public int? play()
```

Comment:
Sorry about that, I could confused between ours and theirs when doing a pull, these are gone now.